src/gateio_new_coins_announcements_bot/announcement_scrapers/binance_scraper.py

```diff
@@ -18,7 +20,17 @@ def fetch_latest_announcement(self):
         """
         logger.debug("Pulling announcement page")
         request_url = self.__request_url()
-        response = self.http_client.get(request_url)
+
+        if rotating_proxy_is_ready():
+            proxy = get_proxy()
+            logger.debug(f"Using proxy: {proxy}")
+            try:
+                response = self.http_client.get(request_url, proxies={"http": "socks5://" + proxy})
```

Comment:
Another point we should consider is what happens when the request via the proxy fails or returns status code 429.
In that case we probably wanna remove the proxy or don't use it for some time (e.g. 5min).
But I don't mind implementing that in another PR considering that this is an opt-in feature via the config anyway.